README.md

```diff
@@ -33,6 +33,7 @@ drozer is open source software, maintained by MWR InfoSecurity, and can be downl
 git clone https://github.com/mwrlabs/drozer/
 cd drozer
 make apks
+source ENVIRONMENT
```

Comment:
as mentioned in my previous comment, this needs to be enabled to help stdeb find the right package structure.